### ISSUE TITLE
Auth - workflow

### ISSUE THREAD

**Issue by igorspasic** [wedeploy/demo-quiz#12]
Here is the auth workflow:

+ user must log in in order to vote.
+ provide option to signup quickly (without sending email, if possible)?
+ in `main.js` there is a method `handleAnswer` that calls method `incrementUserStats` and pass user id to it. Currently, the user id is hardcoded (`userNN`). This change should put a real user id here.

ADDITIONAL

+ make a javascript method that returns list of all users (i.e. userIds).

**Comment by laissandrade** [comment]
I've fixes auth and its working for github, google and email/password.

**Comment by laissandrade** [comment]
I added also redirect on `index.html` to `login.html` if not logged in.

**Comment by cirocosta** [comment]
hey, we can have different directories with `index.html` on each so that the url looks better (`bla.wedeploy.io/signin`, `bla.wedeploy.io/chitchat` .. ) as we already have the rule on nginx to look for the index.htm(l)

**Comment by cirocosta** [comment]
i also prefer this way to have separate `js` files with just the minimal functionality for each section. Wdyt?

**Comment by laissandrade** [comment]
Yes, the js files need to be separate so we can add them step-by-step, and its easy to check. I separated one per file, but we can split it even more inside each page folder. Can you do this tomorrow, @cirocosta?
